README.md

```diff
@@ -1,59 +1,158 @@
+│ --url      -u  TEXT  The OpenSearch instance endpoint minus the http scheme, e.g.                                                   │
+│                      'search-timdex-env-1234567890.us-east-1.es.amazonaws.com'. If not provided, will attempt to get from the       │
+│                      TIMDEX_OPENSEARCH_ENDPOINT environment variable. Defaults to 'localhost'.                                      │
+│ --verbose  -v        Pass to log at debug level instead of info                                                                     │
+│ --help     -h        Show this message and exit.                                                                                    │
+╰─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+╭─ Get cluster-level information ─────────────────────────────────────────────────────────────────────────────────────────────────────╮
+│ ping            Ping OpenSearch and display information about the cluster.                                                          │
+│ indexes         Display summary information about all indexes in the cluster.                                                       │
+│ aliases         List OpenSearch aliases and their associated indexes.                                                               │
+╰─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+╭─ Index management commands ─────────────────────────────────────────────────────────────────────────────────────────────────────────╮
+│ create       Create a new index in the cluster.                                                                                     │
+│ delete       Delete an index.                                                                                                       │
+│ promote      Promote index as the primary alias and add it to any additional provided aliases.                                      │
+│ demote       Demote an index from all its associated aliases.                                                                       │
+╰─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+╭─ Bulk record processing commands ───────────────────────────────────────────────────────────────────────────────────────────────────╮
+│ bulk-index                         Bulk index records into an index.                                                                │
+│ bulk-delete                        Bulk delete records from an index.                                                               │
+╰─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
```

Comment:
Perhaps time will tell if this pattern is a good one... as we do need to keep this up to date with changes, but I must admit I like seeing it in the readme.  This allows seeing the CLI help without cloning, installing, and setting anything up, which is handy.

I spoke with Jeremy awhile back about potentially automating this, which I think could be pretty neat.  Perhaps it would write to a standalone file, something like `docs/cli_help.md`?  But, for the time being, I'm appreciating this in the base README.

Reply:
Automating it sounds epic!